pom.xml

```diff
@@ -145,8 +135,8 @@
                 <jdk.7.excludes>**/SpringBootUtilTest.java</jdk.7.excludes>
             </properties>
         </profile>
-        <profile> 
-            <id>1.8+excludes</id>          
+        <profile>
+            <id>1.8+excludes</id>
             <activation>
                 <jdk>[1.8,10)</jdk>
```

Comment:
Should you remove Java 10 here?

Reply:
`[1.8,10)` means >= Java 8 and < Java 10, so that's what we want.